pkg/metrics/providers/prometheus.go

```diff
@@ -15,6 +15,8 @@ import (
 	flaggerv1 "github.com/weaveworks/flagger/pkg/apis/flagger/v1beta1"
 )
 
+const onlineQuery = "vector(1)"
```

Comment:
```suggestion
const prometheusOnlineQuery = "vector(1)"
```

can you prefix this by `prometheus` ?